client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -381,11 +384,25 @@ class SimplePaymentsDialog extends Component {
 
 		if ( ! shouldQuerySitePlans && isJetpackNotSupported ) {
 			return this.renderEmptyDialog(
-				<Notice
-					status="is-error"
-					text={ translate( 'Please upgrade to Jetpack 5.2 to use Simple Payments feature' ) }
-					onDismissClick={ onClose }
-				/>
+				<EmptyContent
+					className="upgrade-jetpack"
+					illustration="/calypso/images/illustrations/illustration-jetpack.svg"
+					title={ translate( 'Upgrade Jetpack to use Simple Payments feature' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Please upgrade to Jetpack 5.2 to use Simple Payments feature' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).